src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyConsoleLogger.cs

```diff
@@ -101,9 +101,9 @@ private void WriteMessage(LogLevel logLevel, string logName, int eventId, string
                         case LogLevel.Critical:
                             _jsRuntime.InvokeUnmarshalled<string, object>("Blazor._internal.dotNetCriticalError", formattedMessage);
                             break;
-                        default: // LogLevel.None or invalid enum values
-                            Console.WriteLine(formattedMessage);
-                            break;
+                        default: // invalid enum values
+                            Debug.Assert(logLevel != LogLevel.None, "This method is never called with LogLevel.None.");
```

Comment:
```suggestion
                            _jsRuntime.InvokeVoid("console.log", formattedMessage);
```

If we can avoid a breaking change here, it would be great. The above would be an adequate substitute.